google/cloud/security/common/data_access/dao.py

```diff
@@ -41,6 +41,8 @@
     'raw_project_iam_policies':
         create_tables.CREATE_RAW_PROJECT_IAM_POLICIES_TABLE,
     'raw_org_iam_policies': create_tables.CREATE_RAW_ORG_IAM_POLICIES_TABLE,
+
+    'violations': create_tables.CREATE_VIOLATIONS_TABLE,
```

Comment:
nit: would it be worth adding a prefix to the table to denote that it's scanner doing this?